.github/workflows/codespell.yml

```diff
@@ -0,0 +1,16 @@
+# GitHub Action to automate the identification of common misspellings in text files.
```

Comment:
please remove .github directory, we don't want upstream automated workflows.

Reply:
I was trying to see, how did this came back when I clearly picked https://github.com/openshift/azure-disk-csi-driver/pull/75/commits/a063e77c1e3d0de1fc4ab5dec5ccdda53bd990d2 commit. But then I think, this directory came back because two more files were added in upstream, which our downstream PR didn't remove. I will make a new commit for it.

Reply:
Fixed.